src/index.html

```diff
@@ -3,10 +3,42 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="/src/styles/main.scss">
+    <title>31-Day Calendar</title>
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-tue">
+      <div class="days"></div>
```

Comment:
It should be `calendar_day` instead, as days aren't blocks themself and can't be used outside `calendar`